docs/README.md

```diff
@@ -43,7 +43,6 @@ Run these commands
 
 ```bash
 git submodule update --init --recursive
-pip install -e . -r requirements/pytorch/docs.txt
```

Comment:
Why aren't we keeping this and add pandoc here?
Would reduce risk that the text gets outdated when requirements change in the future.

Reply:
My thinking was that if the build commands are ever changed, it'll probably be changed where it's run by CI/CD before it's changed in the docs. So it'd better to have the Makefile be the single source of truth to prevent conflicting instructions. Then only build requirements that are not part of the Makefile (like installing pandoc) need mentioning in the docs.